mappings/net/minecraft/recipe/Ingredient.mapping

```diff
@@ -11,19 +11,26 @@ CLASS ayt net/minecraft/recipe/Ingredient
 		METHOD a getStacks ()Ljava/util/Collection;
 		METHOD b toJson ()Lcom/google/gson/JsonObject;
 	FIELD a EMPTY Layt;
+	FIELD b isEntryNotEmpty Ljava/util/function/Predicate;
```

Comment:
If static final, this should be CAPS_UNDERSCORED like so. Also, just entryNotEmpty?

Reply:
It's a static final Predicate object, but it's only ever used as a callable (like a static method) and never passed around like a constant. I have absolutely no idea why it isn't written as a static method.